server/middleware/auth.coffee

```diff
@@ -179,7 +179,13 @@ module.exports =
     
     activity = req.user.trackActivity 'login', 1
     yield req.user.update {activity: activity}
-    res.redirect '/'
```

Comment:
No login clever test :( [Added a couple](https://github.com/codecombat/codecombat/commit/54484ea4d4e4ac4601f6ac370b8332dc803820a5#diff-061bf41f4b40f427d638fcbab5a5b0d6R322) to make sure this change doesn't break clever integration.